docs/dev/cut-a-new-release.md

```diff
@@ -204,3 +205,23 @@ If you need to release a hotfix for a previously released version (that is: not
     * This is important so that no changes are lost.
     * This might have merge conflicts with the `master` branch, which you need to fix before it is merged.
 8. Once the above pull request lands, delete the support branch.
+
+## Upgrading android-components to a new version of Glean
+
+On Android, many users of Glean import it through its wrapper in [`android-components`](https://github.com/mozilla-mobile/android-components).
+Therefore, when a new Glean release is made, `android-components` must also be updated.
+
+After following one of the above instructions to make a Glean release:
+
+1) Ensure that CI has completed and the artifacts have published to [the Glean GitHub releases page](https://github.com/mozilla/glean/releases/).
+
+2) Publish Glean to [Mozilla's Maven repository](https://maven.mozilla.org/). 
+   This is currently a manual step that can only performed by individuals with the necessary credentials.
+   Ask in `#releaseduty-mobile` on Mozilla's internal Slack for assistance.
+   
+3) Create a pull request against `android-components` to update the Glean version with the following changes:
+   
+   - The Glean version is updated in the `mozilla_glean` variable in the `buildSrc/src/main/java/Dependencies.kt` file.
+   
+   - The relevant parts of the Glean changelog copied into the `android-components` changelog.
```

Comment:
```suggestion
   - The relevant parts of the Glean changelog copied into the [`android-components` changelog](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md), in the top section with the`-SNAPSHOT` suffix in the title.
```

Reply:
That's only relevant if they are releasing snapshots, which IIUC is not the long term plan.  I'd be fine with just saying "top section" here, though.

Reply:
Sounds good to me!